<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1911</width>
      <height>1031</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pb_twitch">
         <property name="font">
          <font>
           <family>DejaVu Sans</family>
           <pointsize>42</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Twitch</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_youtube">
         <property name="font">
          <font>
           <family>DejaVu Sans</family>
           <pointsize>42</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Youtube</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_tv">
         <property name="font">
          <font>
           <family>DejaVu Sans</family>
           <pointsize>42</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Television</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_apps">
         <property name="font">
          <font>
           <family>DejaVu Sans</family>
           <pointsize>42</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Applications</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="pte_debug">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lst_stuff">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>42</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
